src/System.Runtime/tests/Performance/Perf.HashCode.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+
+namespace System.Tests
+{
+    public class Perf_HashCode
+    {
+        [Benchmark]
+        public void Add_()
```

Comment:
Why the trailing underscore?

Reply:
Copy/paste (have a look at Perf.Int32). I thought it might be part of some convention.

Removed.